lang/en/format_learningmap.php

```diff
@@ -0,0 +1,42 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * English language pack for Learningmap
+ *
+ * @package    format_learningmap
+ * @category   string
+ * @copyright  2024 ISB Bayern
+ * @author Stefan Hanauska <stefan.hanauska@csg-in.de>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$string['close'] = 'Close';
+$string['completionrequired'] = 'Completion tracking needs to be enabled to use this course format';
+$string['error_nolearningmap'] = 'This course is not ready yet.';
+$string['hidefromothers'] = 'hide';
+$string['mainlearningmap'] = 'Main learningmap';
+$string['newsection'] = 'New section';
+$string['nolearningmap'] = 'No learningmap in the course';
+$string['nolearningmapnotification'] = 'There is no learningmap in this course. This is no problem, if the course is just being built. It will be unaccessible for students until a learningmap is created.';
+$string['plugin_description'] = 'The course page is replaced by a learning map.';
+$string['pluginname'] = 'Learningmap';
+$string['privacy:metadata'] = 'The Learningmap plugin doesn\'t store any personal data.';
+$string['section0name'] = 'General';
+$string['sectionname'] = 'Section';
+$string['showfromothers'] = 'show';
```

Comment:
until this is being published via moodle plugins directory, could you please also provide a German lang file?